### PR TITLE
khadri111-modified files

### PR DESCRIPTION
**Pull request by sgreddy123**

### CODE REVIEW

File: src/main/java/com/khadri/shopping/amazon/AmazonDriver.java [hunk 1]
```diff
@@ -16,13 +16,13 @@ public static void main(String[] args) {
 
 		do {
 
-			System.out.println("########## Home Page ##########");
```
Comment (khadri2021) [inline src/main/java/com/khadri/shopping/amazon/AmazonDriver.java:19]: hash symbols need arrangae

Comment (sgreddy123) [reply src/main/java/com/khadri/shopping/amazon/AmazonDriver.java:19]: it is fine

**Review by khadri2021 — APPROVED** [review]

File: src/main/java/com/khadri/shopping/amazon/AmazonDriver.java [hunk 2]
```diff
@@ -16,13 +16,13 @@ public static void main(String[] args) {
...
 			System.out.println(" 1.Mens_wear ");
 			System.out.println(" 2.Kids_wear ");
 			System.out.println(" 3.Womens_wear ");
 			System.out.println(" 4.Previous ");
 
-			System.out.println("#####Choose any one Category:(1,2,3)##### ");
+			System.out.println("#########Choose any one Category:(1,2,3)######### ");
 			System.out.println("=============================");
 			int option = scanner.nextInt();
 			switch (option) {
```
Comment (manvithagali52) [inline src/main/java/com/khadri/shopping/amazon/AmazonDriver.java:28]: you have to rework for this

Comment (sgreddy123) [reply src/main/java/com/khadri/shopping/amazon/AmazonDriver.java:28]: ok

**Review by manvithagali52 — APPROVED** [review]

**Pull request merged**